packages/frameworks/src/frameworks.ts

```diff
@@ -24,16 +24,14 @@ export const frameworks = [
     logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/blitz.svg',
     tagline: 'Blitz.js: The Fullstack React Framework',
     description:
-      'A brand new Blitz.js app - the result of running `npx blitz new`.',
+      'A brand new Blitz.js app - the result of running `npx blitz@0.45.4 new`.',
```

Comment:
Lets revert the description since the version is irrelevant

Reply:
Hey @styfle, We have to set the version otherwise it will installed the latest version of blitz which is `2.0.0-beta.17`.